source/sphinx_extensions/plugin_directory/templates/plugin.rst

```diff
@@ -6,6 +6,10 @@
    <table class="table plugin-info">
      <tbody>
        <tr>
+         <th scope="row">Description</th>
+         <td>{{ plugin.description|urlize }}</td>
```

Comment:
Done.  Yeah I also noticed that `urlize` can be strange sometimes.  Looks to me like it's working now that I changed the code to reflect the structure used in `Support` and `Citation` per your suggestion.